pyproject.toml

```
@@ -63,4 +66,69 @@ dev = [
     "types-pyyaml>=6.0.12.20250516",
     "google-re2-stubs",
     "sphinxcontrib-plantuml>=0.30",
+    "google-cel-spec-protocolbuffers-python==32.0.0.1.20250512201741+dffb8c8cf781",
```

nit to make the versions match:
```suggestion
    "google-cel-spec-protocolbuffers-python==32.1.0.1.20250512201741+dffb8c8cf781",
```